### DISCUSSION TITLE
Support custom events as flow trigger

### DISCUSSION THREAD

**Discussion by ashitikov** [directus/directus#14464] [Feature Requests]
Hello! I'm new to directus, and I'm trying to figure out whether I can use custom event as flow's trigger? 
Here I can set the scope (which event I wanna listen). Suppose I publish custom event using event emitter, how can I setup a trigger, based on that event?
<img width="860" src="https://user-images.githubusercontent.com/8819544/179307957-c8480cbf-3111-4fa3-ae6f-8fce2bd9c067.png">

**Comment by rijkvanzanten** [comment]
I think it should already work if you use the "Edit Raw Value" to set the Scope value to a custom event name, though I've never tried that myself. I'll covert this discussion to be the feature request to properly support this workflow

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
